src/java/htsjdk/samtools/util/BlockCompressedInputStream.java

```diff
@@ -354,53 +366,76 @@ private static boolean isValidBlockHeader(final byte[] buffer) {
                 buffer[12] == BlockCompressedStreamConstants.BGZF_ID1 &&
                 buffer[13] == BlockCompressedStreamConstants.BGZF_ID2);
     }
-
+    
     private void readBlock()
```

Comment:
Add a comment what `readBlock` does (now that there is a `readBlock()` and `nextBlock()` we need better comments